package.json

```diff
@@ -1,102 +1,53 @@
 {
     "name": "angular-auth-oidc-client",
-    "private": true,
-    "version": "9.0.3",
-    "description": "An OpenID Connect Code Flow with PKCE,Implicit Flow client for Angular",
-    "main": "./bundles/angular-auth-oidc-client.umd.js",
-    "module": "./fesm5/angular-auth-oidc-client.js",
-    "es2015": "./fesm2015/angular-auth-oidc-client.js",
-    "esm5": "./esm5/angular-auth-oidc-client.js",
-    "esm2015": "./esm2015/angular-auth-oidc-client.js",
-    "fesm5": "./fesm5/angular-auth-oidc-client.js",
-    "fesm2015": "./fesm2015/angular-auth-oidc-client.js",
-    "typings": "./angular-auth-oidc-client.d.ts",
+    "version": "0.0.0",
     "scripts": {
-        "build": "node build.js",
-        "test": "karma start",
-        "test:watch": "karma start karma.conf.js --single-run false",
-        "pack:lib": "npm run build && npm pack ./dist",
-        "publish:lib": "npm run build && npm publish ./dist",
-        "publish:lib:next": "npm run build && npm publish --tag next ./dist",
-        "compodoc": "compodoc -p tsconfig.json",
-        "compodoc:serve": "compodoc -s"
-    },
-    "author": "",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/damienbod/angular-auth-oidc-client.git"
+        "ng": "ng",
+        "watch-lib": "ng build angular-auth-oidc-client --watch",
+        "build-lib": "ng build angular-auth-oidc-client",
+        "test-lib": "ng test angular-auth-oidc-client",
+        "test-lib-ci": "ng test angular-auth-oidc-client --watch=false --browsers=ChromeHeadless",
+        "lint-lib": "ng lint angular-auth-oidc-client"
     },
-    "bugs": {
-        "url": "https://github.com/damienbod/angular-auth-oidc-client/issues"
-    },
-    "homepage": "https://github.com/damienbod/angular-auth-oidc-client",
-    "keywords": [
-        "angular",
-        "oidc",
-        "oauth2",
-        "openid",
-        "security",
-        "typescript"
-    ],
-    "license": "MIT",
+    "private": true,
     "dependencies": {
+        "@angular/animations": "~7.2.0",
```

Comment:
we supported Angular 6 up to now, need to know if this is a breaking change

Reply:
This is not the lib, this is the project. The peerdependency describes what we support.